src/components/Loading/Loading.tsx

```diff
@@ -5,19 +5,29 @@ import { CircularProgress as theme } from '../../theme/components/external/Circu
 interface Props {
 
   /** Is data loading. */
-  isLoading: boolean;
+  readonly isLoading: boolean;
 
   /** Children node. */
-  children?: ReactNode;
+  readonly children?: ReactNode;
+
+  /** Whether loader should take full height. */
+  readonly fullHeight?: boolean;
```

Comment:
Again, don't forget about adding prefixes for `boolean` props, consts, variables and etc. I think it's common sense and logically correct + you can check props of `Chakra UI` components, `Chakra UI` devs stick to such namings too 🙄